app/models/membership-role.js

```diff
@@ -40,11 +40,13 @@ export default class MembershipRoleModel extends Model {
   @attr('string') label;
 
   @belongsTo('concept', {
+    async: true,
     inverse: null,
   })
   topConceptOf;
 
   @belongsTo('membership-role', {
+    async: true,
```

Comment:
I took another look at it and it seems I ran into a bug that was already present, so it has nothing to do with this change. Apologies for pointing fingers too quickly :blush: 

For completeness: the bug occurs when adding and removing a new related organization without filling in the data. In that case when pressing save a `TypeError` occurs because, presumably, the removal of the empty membership does not work correctly. Bug logged as OP-3371 which we'll resolve as part of the whole membership refactoring.